tests/ci/README.md

```diff
@@ -0,0 +1,82 @@
+# CI for AWS-LC
+We use prebuilt docker images for all of our builds for speed and consistency.
+
+## Setup
+ To setup the images for local testing or testing in your own AWS account see
+the platform specific `README` in docker_images/*.
+
+Once you have the docker images uploaded to AWS Elastic Container Registry you
+can setup the AWS CodeBuild projects that use the custom image with the
+appropriate buildspec files in codebuild/*.
+
+## Local testing
+The best way to test AWS-LC locally is to use the same Docker containers AWS
+CodeBuild uses.
+1. Install Docker
+2. Navigate to your AWS-LC project directory
+3. Build the docker image you want to test
+4. Run the docker image
+   *   Use `-v` to pass a volume from the host to the container, `pwd`:`pwd`
+       mounts the same path on the host to the container. This ensures the
+       container will build and test your exact working state.
+   *  Use `-w` to change to that directory inside the container after launching
+      it
+5. Run the build
+
+For example testing x86-64 Ubuntu 19.10 clang 9x:
+```
+$ cd $AWS_LC_PROJECT_ROOT
+$ docker build -t ubuntu-19.10:clang-9x tests/ci/docker_images/linux-x86/ubuntu-19.10_clang-9x/
+$ docker run -v `pwd`:`pwd` -w `pwd` -it ubuntu-19.10:clang-9x
+$ ./tests/ci/run_posix_tests.sh
+```
+
+## Test locations
+### Unit tests
+Runs `test_build_dir/crypto/crypto_test` for:
```

Comment:
We run all_tests.go which then runs everything in https://github.com/awslabs/aws-lc/blob/main/third_party/boringssl/util/all_tests.json

Reply:
Changed.